Styles/style.css

```diff
@@ -0,0 +1,68 @@
+@import url(reset.css);
+
+.block {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.red {
+  height: 20vh;
```

Comment:
common styles like `height, width, transition-duration` you can add to class block, and it will be used for all 5 block

red green and other classes are used for adding background color